src/pages/myAdmin/guides/codeBase/usingWithJavascript.tsx

```diff
@@ -59,11 +60,10 @@ myAdminApi().call('Authenticate', logonParams, function(user) {
       <code className="small-code-sample">logonParams</code> object and provides
       a callback function to be executed following a successful login. The
       callback function receives an ApiUser object which contains, among other
-      properties, the user's API key (userId) and session ID. See the{" "}
-      <a href="../../api/reference"> {/* ToDo: fix link */}Reference</a>{" "}
+      properties, the user's API key (userId) and session ID. See the reference{" "}
       documentation for more information on the{" "}
-      <code className="small-code-sample">Authenticate</code> method and the{" "}
-      <code className="small-code-sample">ApiUser</code> object.
+      <Link to="/myAdmin/apiReference/methods">Authenticate</Link> method and
+      the <Link to="/myAdmin/apiReference/objects">ApiUser</Link> object.
```

Comment:
I know the nav bar routes to this path but maybe we need to add a TODO here to confirm once the page is added (similar to what is seen in the MyAdminNavBarItems.ts